hlc/plan/planner.py

```diff
@@ -0,0 +1,175 @@
+FORWARD = (0, 1)
+RIGHT_FORWARD = (1, 1)
+RIGHT = (1, 0)
+RIGHT_BACKWARD = (1, -1)
+BACKWARD = (0, -1)
+LEFT_BACKWARD = (-1, -1)
+LEFT = (-1, 0)
+LEFT_FORWARD = (-1, 1)
+
+def add_pos_tuple(t1, t2):
+    new_x = t1[0] + t2[0]
+    new_y = t1[1] + t2[1]
+    return (new_x, new_y)
+
+def sub_pos_tuple(t1, t2):
+    new_x = t1[0] - t2[0]
+    new_y = t1[1] - t2[1]
+    return (new_x, new_y)
+
+
+def layer_pos(grid_dimension, layer):
+    left_layer = []
+    right_layer = []
+    top_layer = []
+    bottom_layer = []
+
+    for i in range(grid_dimension[1]):
+        left_layer.append((layer, i))
+
+    for j in range(grid_dimension[1]):
+        right_layer.append(grid_dimension[1] - layer, j)
+
+    for k in range(grid_dimension[0]):
+        top_layer.append(k, grid_dimension[0] - layer)
+
+    for l in range(grid_dimension[0]):
+        bottom_layer.append(l, layer)
```

Comment:
This function is never called and useless as long as it doesn't return anything.

Reply:
that's the preparation for the obstacle avoidance function, but I'm gonna add returning the values 😬